{"uid":"21a01d242b2dfa94","name":"Test get entity","fullName":"tests.api.test_api.TestEntityAPI#test_get_entity","historyId":"57eefbb9ff7015b44fba60a77b0003f9","time":{"start":1727257960489,"stop":1727257960489,"duration":0},"description":"Verify that a specific entity can be retrieved","descriptionHtml":"<p>Verify that a specific entity can be retrieved</p>\n","status":"broken","statusMessage":"requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://a031-95-32-133-30.ngrok-free.app/api/create","statusTrace":"api_client = <services.entity.api_client.APIClient object at 0x7fa1c7ee81a0>\n\n    @pytest.fixture\n    def entity_id(api_client: APIClient) -> str:\n        \"\"\"Fixture to create and yield a new entity ID.\"\"\"\n>       entity_id = api_client.create_entity()\n\nconftest.py:18: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nservices/entity/api_client.py:21: in create_entity\n    response.raise_for_status()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <Response [404]>\n\n    def raise_for_status(self):\n        \"\"\"Raises :class:`HTTPError`, if one occurred.\"\"\"\n    \n        http_error_msg = \"\"\n        if isinstance(self.reason, bytes):\n            # We attempt to decode utf-8 first because some servers\n            # choose to localize their reason strings. If the string\n            # isn't utf-8, we fall back to iso-8859-1 for all other\n            # encodings. (See PR #3538)\n            try:\n                reason = self.reason.decode(\"utf-8\")\n            except UnicodeDecodeError:\n                reason = self.reason.decode(\"iso-8859-1\")\n        else:\n            reason = self.reason\n    \n        if 400 <= self.status_code < 500:\n            http_error_msg = (\n                f\"{self.status_code} Client Error: {reason} for url: {self.url}\"\n            )\n    \n        elif 500 <= self.status_code < 600:\n            http_error_msg = (\n                f\"{self.status_code} Server Error: {reason} for url: {self.url}\"\n            )\n    \n        if http_error_msg:\n>           raise HTTPError(http_error_msg, response=self)\nE           requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://a031-95-32-133-30.ngrok-free.app/api/create\n\n/opt/hostedtoolcache/Python/3.12.6/x64/lib/python3.12/site-packages/requests/models.py:1024: HTTPError","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":5,"retriesStatusChange":false,"beforeStages":[{"name":"_skip_sensitive","time":{"start":1727257960489,"stop":1727257960490,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"entity_id","time":{"start":1727257960490,"stop":1727257960526,"duration":36},"status":"broken","statusMessage":"requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://a031-95-32-133-30.ngrok-free.app/api/create\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.12.6/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/opt/hostedtoolcache/Python/3.12.6/x64/lib/python3.12/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.6/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.6/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.6/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/runner/work/TestNest/TestNest/conftest.py\", line 18, in entity_id\n    entity_id = api_client.create_entity()\n                ^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.6/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/TestNest/TestNest/services/entity/api_client.py\", line 21, in create_entity\n    response.raise_for_status()\n  File \"/opt/hostedtoolcache/Python/3.12.6/x64/lib/python3.12/site-packages/requests/models.py\", line 1024, in raise_for_status\n    raise HTTPError(http_error_msg, response=self)\n","steps":[{"name":"Create entity","time":{"start":1727257960490,"stop":1727257960525,"duration":35},"status":"broken","statusMessage":"requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://a031-95-32-133-30.ngrok-free.app/api/create\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.12.6/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/TestNest/TestNest/services/entity/api_client.py\", line 21, in create_entity\n    response.raise_for_status()\n  File \"/opt/hostedtoolcache/Python/3.12.6/x64/lib/python3.12/site-packages/requests/models.py\", line 1024, in raise_for_status\n    raise HTTPError(http_error_msg, response=self)\n","steps":[],"attachments":[{"uid":"1b2ab74aa7bc4bcf","name":"API Response","source":"1b2ab74aa7bc4bcf.txt","type":"text/plain","size":2430},{"uid":"200ceb4149390384","name":"Status Code","source":"200ceb4149390384.txt","type":"text/plain","size":3},{"uid":"4ecd78ab3408ac85","name":"Headers","source":"4ecd78ab3408ac85.txt","type":"text/plain","size":197}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":3,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":1,"attachmentsCount":3,"hasContent":true,"attachmentStep":false},{"name":"api_client","time":{"start":1727257960490,"stop":1727257960490,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"Verify that a specific entity can be retrieved","status":"broken","statusMessage":"requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://a031-95-32-133-30.ngrok-free.app/api/create","statusTrace":"api_client = <services.entity.api_client.APIClient object at 0x7fa1c7ee81a0>\n\n    @pytest.fixture\n    def entity_id(api_client: APIClient) -> str:\n        \"\"\"Fixture to create and yield a new entity ID.\"\"\"\n>       entity_id = api_client.create_entity()\n\nconftest.py:18: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nservices/entity/api_client.py:21: in create_entity\n    response.raise_for_status()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <Response [404]>\n\n    def raise_for_status(self):\n        \"\"\"Raises :class:`HTTPError`, if one occurred.\"\"\"\n    \n        http_error_msg = \"\"\n        if isinstance(self.reason, bytes):\n            # We attempt to decode utf-8 first because some servers\n            # choose to localize their reason strings. If the string\n            # isn't utf-8, we fall back to iso-8859-1 for all other\n            # encodings. (See PR #3538)\n            try:\n                reason = self.reason.decode(\"utf-8\")\n            except UnicodeDecodeError:\n                reason = self.reason.decode(\"iso-8859-1\")\n        else:\n            reason = self.reason\n    \n        if 400 <= self.status_code < 500:\n            http_error_msg = (\n                f\"{self.status_code} Client Error: {reason} for url: {self.url}\"\n            )\n    \n        elif 500 <= self.status_code < 600:\n            http_error_msg = (\n                f\"{self.status_code} Server Error: {reason} for url: {self.url}\"\n            )\n    \n        if http_error_msg:\n>           raise HTTPError(http_error_msg, response=self)\nE           requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://a031-95-32-133-30.ngrok-free.app/api/create\n\n/opt/hostedtoolcache/Python/3.12.6/x64/lib/python3.12/site-packages/requests/models.py:1024: HTTPError","steps":[],"attachments":[{"uid":"9770095aecbad0e8","name":"log","source":"9770095aecbad0e8.txt","type":"text/plain","size":1571},{"uid":"af28ac8d0fa6eed6","name":"stderr","source":"af28ac8d0fa6eed6.txt","type":"text/plain","size":1254}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":2,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"epic","value":"Entity Management"},{"name":"feature","value":"CRUD Operations"},{"name":"tag","value":"api"},{"name":"parentSuite","value":"tests.api"},{"name":"suite","value":"test_api"},{"name":"subSuite","value":"TestEntityAPI"},{"name":"host","value":"fv-az1490-494"},{"name":"thread","value":"2253-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.api.test_api"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"128d704cf349a888","status":"broken","statusDetails":"requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://a031-95-32-133-30.ngrok-free.app/api/create","time":{"start":1727257955434,"stop":1727257955434,"duration":0}},{"uid":"9920e58e01ab976b","status":"broken","statusDetails":"requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://a031-95-32-133-30.ngrok-free.app/api/create","time":{"start":1727257950383,"stop":1727257950383,"duration":0}},{"uid":"97d8ea5ff97efddb","status":"broken","statusDetails":"requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://a031-95-32-133-30.ngrok-free.app/api/create","time":{"start":1727257945328,"stop":1727257945328,"duration":0}},{"uid":"ef5dec893641cb52","status":"broken","statusDetails":"requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://a031-95-32-133-30.ngrok-free.app/api/create","time":{"start":1727257940275,"stop":1727257940275,"duration":0}},{"uid":"758428ec15ef39d6","status":"broken","statusDetails":"requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://a031-95-32-133-30.ngrok-free.app/api/create","time":{"start":1727257935221,"stop":1727257935221,"duration":0}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":1,"skipped":0,"passed":3,"unknown":0,"total":4},"items":[{"uid":"3cecf44e1aa46b50","reportUrl":"https://ZhikharevAl.github.io/TestNest/7//#testresult/3cecf44e1aa46b50","status":"passed","time":{"start":1727235455823,"stop":1727235456262,"duration":439}},{"uid":"a1b4518041e8539d","reportUrl":"https://ZhikharevAl.github.io/TestNest/6//#testresult/a1b4518041e8539d","status":"passed","time":{"start":1727234276300,"stop":1727234276690,"duration":390}},{"uid":"28db050cbaed3556","reportUrl":"https://ZhikharevAl.github.io/TestNest/5//#testresult/28db050cbaed3556","status":"passed","time":{"start":1727230686737,"stop":1727230687095,"duration":358}}]},"tags":["api"]},"source":"21a01d242b2dfa94.json","parameterValues":[]}